src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -368,6 +357,14 @@ object EntityClient {
 
   sealed trait SecurityTag {
     override def toString: String = getClass.getSimpleName.dropRight(1).toLowerCase
+
+  }
+  object SecurityTag {
+    def fromString(securityTagString: String): Option[SecurityTag] = securityTagString match {
+      case "open"   => Option(Open)
+      case "closed" => Option(Closed)
+      case _        => None
```

Comment:
Should this case be an Exception? From what I understand, we're only using open and closed (at least for now). I ask because one of the test documents had a securityTag of "public" (which was wrong) which got caught by the upsert Lambda but might not get flagged elsewhere

Reply:
We've tried throwing exceptions for bad data before and the Preservica will send something weird our way for no reason. As you say, it got caught further up so I think it's fine here for the client to return whatever is or isn't there and we deal with it later. We can always change our minds.